types/query/pagination.go

```diff
@@ -24,6 +24,11 @@ const DefaultLimit = 100
 var PaginationMaxLimit uint64 = math.MaxUint64
 
 // ParsePagination validate PageRequest and returns page number & limit.
+// Note: cometBFT enforces a maximum query limit of 100 to avoid node overload.
```

Comment:
this is also true for our pagination right?

Reply:
Yes, this method is used in methods that use CometBFT behind it, so the behavior is the same

Reply:
if we enforce a max of 100 as well, then we should mention this as well